config/proto/targets.proto

```diff
@@ -34,15 +47,17 @@ message TargetsDef {
 
 // REQUIRED, when using Ceph as the target system.
 // Comparable functionality is handled in the Go client library for GCS.
-message CephConnectionDetails {
-  int32 port = 1;  // Port for connecting to Ceph S3 API
-  
+message ConnectionDetails {
+  // REQUIRED for Ceph S3 API
```

Comment:
Stupid question maybe, but do you not need to give a hostname for ceph?

Reply:
I think this should be contained in the target_url for connecting to the API?

Reply:
cual beans